docs/_config.yml

```diff
@@ -2,6 +2,9 @@
 title: Solace Samples SEMP
 baseurl: "/solace-samples-semp" # the subpath of your site, e.g. /solace-samples-java
 repository: https://github.com/SolaceSamples/solace-samples-semp
+repository-java: https://github.com/SolaceSamples/solace-samples-semp/Java
```

Comment:
Could we rename the subdirectory to `java`. Convention is not to use cases up until this point.